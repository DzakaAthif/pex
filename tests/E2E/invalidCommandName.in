GRAB 0 Avocado 50 100;
TOSS 0 Avocado 50 100;
123 0 Avocado 50 100;
0 Avocado 50 100;
SELL1 0 Avocado 50 100;
BUY1 0 Avocado 50 100;
BUY 0 Avocado 50 100;
AMEND1 0 60 110;
CANCEL1 0;