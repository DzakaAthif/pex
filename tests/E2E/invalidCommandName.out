[PEX] Starting
[PEX] Trading 2 products: Wine Avocado
[PEX] Created FIFO /tmp/pe_exchange_0
[PEX] Created FIFO /tmp/pe_trader_0
[PEX] Starting trader 0 (./trader1)
[PEX] Connected to /tmp/pe_exchange_0
[PEX] Connected to /tmp/pe_trader_0
[PEX] [T0] Parsing command: <GRAB 0 Avocado 50 100>
[PEX] [T0] Parsing command: <TOSS 0 Avocado 50 100>
[PEX] [T0] Parsing command: <123 0 Avocado 50 100>
[PEX] [T0] Parsing command: <0 Avocado 50 100>
[PEX] [T0] Parsing command: <SELL1 0 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY1 0 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 100>
[PEX]	--ORDERBOOK--
[PEX]	Product: Wine; Buy levels: 0; Sell levels: 0
[PEX]	Product: Avocado; Buy levels: 1; Sell levels: 0
[PEX]		BUY 50 @ $100 (1 order)
[PEX]	--POSITIONS--
[PEX]	Trader 0: Wine 0 ($0), Avocado 0 ($0)
[PEX] [T0] Parsing command: <AMEND1 0 60 110>
[PEX] [T0] Parsing command: <CANCEL1 0>
[PEX] Trader 0 disconnected
[PEX] Trading completed
[PEX] Exchange fees collected: $0
